xdrip-get-entries.sh

```diff
@@ -717,7 +717,7 @@ function log_g5_csv()
 # if tx state or status changed, then post a note to NS
 function process_announcements()
 {
-  if [ "$mode" == "Stopped" ]; then
+  if [ "$state" == "Stopped" ]; then
```

Comment:
I suggest just deleting this one because I think it is good for announcements to go out even if the state is stopped for both modes.